### DISCUSSION TITLE
`packeto-community` project not found on GCR (Google Container Registry)

### DISCUSSION THREAD

**Discussion by ghost** [paketo-buildpacks/feedback#253] [Q&A]
Hi team, I just found that I couldn't pull images under `gcr.io/packeto-community/*`

Has it been removed? Should we use `docker.io` container registry?

Thanks! 🙏 

https://console.cloud.google.com/gcr/images/paketo-community

**Comment by dmikusa** [comment]
tl;dr - Yes, use docker.io.

------------

Aside from that, were you previously pulling images for paketo-community from GCR? Which images are you specifically trying to get?

**Reply by dmikusa** [reply]
An update on this. We're investigating an issue with the Paketo Community repo on GCR. For the time being, please use `docker.io`. We publish to both places, so you should get the same images there. Sorry for the inconvenience.

**Reply by ghost** [reply]
I was trying to pull the `build-plan` image from GCR. I just switched to `docker.io` which worked.
